library/core/src/num/int_macros.rs

```diff
@@ -918,6 +918,32 @@ macro_rules! int_impl {
             }
         }
 
+        /// Saturating integer division. Computes `self / rhs`, saturating at the
+        /// numeric bounds instead of overflowing.
+        ///
+        /// # Examples
+        ///
+        /// Basic usage:
+        ///
+        /// ```
+        ///
```

Comment:
Is there any way to put some tests in here? The tests from within `Saturating` only apply to signed types...

Reply:
5 / 2 = 2
MAX / -1 = MIN+1
MIN / -1 = MAX
(1 / 0 = panic)

Reply:
Oh, I didn't realize this is the impl for only signed types... woopsie :)